zng/docs/spec.md

```diff
@@ -3,18 +3,19 @@
 > ### Note: This specification is in BETA development.
 > We hope that no backward incompatible changes will be made during
 > the BETA phase.  We plan to
-> declare the specification stable and finalized in spring 2021.
+> declare the specification stable and finalized in Spring, 2021.
```

Comment:
Neither capitalization nor comma is conventional here.

Reply:
Ah, right you are. I mistakenly thought it was a similar caps/comma rule as when writing month/day/year such as "October 21, 2020". But I see now I was wrong about that and have fixed.